string[] reference_types
string[] target_types
---
semap_msgs/ObjectPair[] pairs
